packages/wdio-logger/tests/node.test.ts

```diff
@@ -130,12 +140,14 @@ describe('wdio-logger node', () => {
         const logInfoSpy = jest.spyOn(fs, 'createWriteStream')
         const logCacheAddSpy = jest.spyOn(Set.prototype, 'add')
         const logCacheForEachSpy = jest.spyOn(Set.prototype, 'forEach')
-        let writableBuffer = null
-        logInfoSpy.mockImplementation((path) => ({
-            path,
+        let writableBuffer: any = null
+        logInfoSpy.mockImplementation((path: fs.PathLike): fs.WriteStream => ({
```

Comment:
Why first assign fs.PathLike and then cast it to a string type when it's only used here?

Reply:
this is how types are defined in Node for some reason, e.g.
```ts
function createWriteStream(path: fs.PathLike, ....)
```
Whereas `WriteStream` is defined as
```ts
    class WriteStream extends stream.Writable {
        close(): void;
        bytesWritten: number;
        path: string | Buffer;
```